TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/wwwroot/Styles/site.scss

```diff
@@ -72,9 +72,18 @@
   max-width: none;
 }
 
-.trs-radios__divider--heading-s {
+.trs-checkboxes_hint {
+  padding-top: 1rem;
+}
+
+.trs-radios__divider--heading-s,
+.trs-checkboxes__divider--heading-s {
   @include govuk-text-colour;
   @include govuk-font($size: 19, $weight: bold);
   width: 400px;
   text-align: left;
 }
+
+.govuk-checkboxes__divider.trs-checkboxes__divider--spaced:not(:first-of-type) {
```

Comment:
Do we need to create another style called trs-checkboxes__divider--spaced?
i.e. can't it just be the default behaviour?
`.trs-checkboxes__divider--heading-s:not(:first-of-type)`

Reply:
I need to apply the trs-checkboxes__divider--heading-s format to all the dividers and use the the :not(:first-of-type) for just the spacing, so I added an extra class. That was how i understood it.

Reply:
oh, I see how to do it - soz